packages/airnode-starter/README.md

```diff
@@ -0,0 +1,193 @@
+never leave your machine.
+
+### 5. Create Airnode secrets
+
+Airnode is configured by two files - `config.json` and `secrets.env`. Run the script below to create the `secrets.env` file with the necessary values.
+
+```sh
+yarn create-airnode-secrets
+```
+
+_Refer to the [documentation](https://docs.api3.org/airnode/next/grp-providers/guides/build-an-airnode/configuring-airnode.html) for more details_
+
+### 6. Build docker artifacts
+
+Our docker images are based on a common container which we call "artifacts". This intermediate container is then used by
+both [deployer](https://github.com/api3dao/airnode/tree/master/packages/deployer) and
+[airnode](https://github.com/api3dao/airnode/tree/master/packages/node). You can build the artifacts container by running:
+
+```sh
+yarn rebuild-artifacts-container
+```
+
+### 7. (Only if deploying to AWS) Build deployer container
+
+```sh
+yarn rebuild-deployer-container
+```
+
+This command will facilitate the previously built artifacts container to build the deployer.
+
+### 8. (Only if deploying to AWS) Deploy Airnode
+
+Now you're ready to deploy Airnode on AWS. Just run:
+
+```sh
+yarn deploy-airnode
+```
+
+This command will just use the [airnode deployer](https://github.com/api3dao/airnode/tree/master/packages/deployer) package. Deployment may take some time so be patient.
+
+### 9. (Only if running Airnode locally) Build Airnode docker container
+
+```sh
+yarn rebuild-airnode-container
+```
+
+This command will facilitate the previously built artifacts container to build the containerized version of Airnode
+which you can run locally.
+
+### 10. (Only if running Airnode locally) Run the Airnode container
+
+```sh
+yarn run-airnode-locally
+```
+
+Runs the previously built version of Airnode container. Note that the containerized version runs a cron job which
+triggers every minute - this means that Airnode logs won't start appearing immediately.
+
+### 11. Deploy a requester
+
+At this point, you have a RRP contract deployed. You also either have a docker running locally or deployed on AWS.
+Airnode is now listening on the events (requests to be made) of the RRP contract. All that is left now, is making a
+request to it. 
+
+The first step is to deploy a [requester](link) contract. Via this contract we will trigger an Airnode request using one
+of our scripts. Run:
+
+```sh
+yarn deploy-requester
+```
+
+### 12. Derive and fund the sponsor wallet
+
+Airnode requests requires a [sponsor](link), which will pay for the response transaction made by Airnode. Each sponsor
+has a dedicated wallet for a given Airnode. This wallet is called a "sponsor wallet" and can be derived from sponsor
+address and Airnode extended public key with the [admin CLI package](link). Refer to the
+[documentation](https://docs.api3.org/airnode/next/grp-developers/requesters-sponsors.html#how-to-derive-a-sponsor-wallet)
+for more details.
+
+Run:
+```sh
+yarn derive-and-fund-sponsor-wallet
```

Comment:
Ran into:
```bash
> yarn derive-and-fund-sponsor-wallet
yarn run v1.22.11
$ ts-node scripts/derive-and-fund-sponsor-wallet.ts
error Command "api3-admin" not found.
```

Likely related to the discussion: https://github.com/api3dao/airnode/pull/571#discussion_r716389369 ? I also ran into the same error in Steps 13 & 14

Reply:
This is strange - after I run `yarn build` from the root directory - I get the `permission denied` error, but the `api3-admin` command is found... (Which I know the reason for).

Reply:
I initially ran `yarn run build` according to the setup instructions and just reran it. No luck. Is there something about paths that I could be missing?

Reply:
@amarthadan Looked at this in the meantime and will create a fix for it.

And I've created a fix for the second issue (permission denied) https://github.com/api3dao/airnode/pull/581

Reply:
I can confirm I no longer get the `"api3-admin" not found` error, but do now get the permission denied error you mention:
```bash
> yarn derive-and-fund-sponsor-wallet
yarn run v1.22.11
$ ts-node scripts/derive-and-fund-sponsor-wallet.ts
/bin/sh: 1: /home/derek/api3/airnode/node_modules/.bin/api3-admin: Permission denied
```

Reply:
https://github.com/api3dao/airnode/pull/583 should fix this.

Reply:
I pulled that `package.json` change in from #583 and ran into the following at `yarn run build` from the `airnode/` root, which appears to be the same error as the [CI is now returning](https://github.com/api3dao/airnode/runs/3768430842):

```bash
@api3/airnode-starter: $ hardhat compile
@api3/airnode-starter: An unexpected error occurred:
@api3/airnode-starter: src/cli.ts(18,20): error TS2345: Argument of type 'unknown' is not assignable to parameter of type 'string'.
@api3/airnode-starter: error Command failed with exit code 1.
@api3/airnode-starter: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run build exited 1 in '@api3/airnode-starter'
error Command failed with exit code 1.
```